### PR TITLE
Add librubberband

### PR DESCRIPTION
**Pull request by wader**
Based on #120

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -307,8 +318,11 @@ RUN \
   libdavs2: env.DAVS2_VERSION, \
   libxavs2: env.XAVS2_VERSION, \
   libmodplug: env.LIBMODPLUG_VERSION, \
-  uavs3d: env.UAVS3D_COMMIT, \
+  libuavs3d: env.UAVS3D_COMMIT, \
```
Comment (wader) [inline Dockerfile:321]: I've used to same naming as ffmpeg here but maybe better to name things similar to what the project uses?

File: Dockerfile [hunk 2]
```diff
@@ -542,6 +556,15 @@ RUN \
...
   make -j$(nproc) install
 
+RUN \
+  wget -O rubberband.tar.bz2 "$RUBBERBAND_URL" && \
+  echo "$RUBBERBAND_SHA256  rubberband.tar.bz2" | sha256sum --status -c - && \
+  tar xf rubberband.tar.bz2 && \
+  cd rubberband-* && \
+  meson -Dno_shared=true -Dfft=fftw build && \
+  ninja -j$(nproc) -vC build install && \
+  echo "Requires.private: fftw3 samplerate" >> /usr/local/lib/pkgconfig/rubberband.pc
```
Comment (wader) [inline Dockerfile:566]: Wonder if this could be reported upstream?

File: Dockerfile (hunk 1 above)
Comment (mathieu-aubin) [reply Dockerfile:321]: i do agree!

Comment (wader) [reply Dockerfile:321]: Use ffmpeg naming or project name? :) think ill merge and we can fix thing later

File: Dockerfile (hunk 2 above)
Comment (wader) [reply Dockerfile:566]: Seems meson pkgconfig has some kind of support for this https://mesonbuild.com/Pkgconfig-module.html but i haven't used it myself. Maybe should just file a issue about it?

File: Dockerfile (hunk 1 above)
Comment (mathieu-aubin) [reply Dockerfile:321]: right, unclear. i meant either way, as long as there is a recognizable standard that makes sense

File: Dockerfile (hunk 2 above)
Comment (mathieu-aubin) [reply Dockerfile:566]: yes, i actually started reading about it a while back today.. i see the build file and somewhat understand the process but i personally can't justify the time to further investigate. Reporting is something to consider

**Pull request merged**